rdm/base/cyber_security.md

```diff
@@ -0,0 +1,12 @@
+---
+category: CYBSEC
+id: CYBSEC-001
+revision: 1
+title: Cyber Security Analysis
+manufacturer_name: {{ system.project_name }}
+---
+
+{{ system.project_name }}
```

Comment:
Is this line intentional?

Reply:
If so, it seems like it will insert the project name at the top of the document.  Other than this, it looks great.

Reply:
No. Typo. Will fix.